### ISSUE TITLE
Tweet importer node

### ISSUE THREAD

**Issue by danielepolencic** [uasabi/noderosso#19]
*TL;DR*: Create a node-red node to import tweets from a CSV file.

A CSV file contains all the tweets due to be published. The current CSV has the following columns:

```
link,total_sources,sources,description,image_1,image_2
```

The file should be ingested according to this logic: https://github.com/learnk8s/lms/blob/master/packages/smartqueue-importer/index.ts#L31-L77

The importer should check:

- The tweet is less than 280 chars. The URL should be shortened when you compute the length: https://github.com/learnk8s/lms/blob/master/packages/smartqueue-importer/index.ts#L129-L138
- The images should be valid images. There should be only 1 GIF or up to 4 images (jpeg, png, jpeg).
- The link is a valid link and there is no other link in the description. https://github.com/learnk8s/lms/blob/master/packages/smartqueue-importer/index.ts#L79-L83

If any of the tweets don't pass the validation, the node should not continue but present the error to the user.

If all tweets are valid, it should template the tweet and generate the following event:

```ts
interface TemplatedTweet {
  topic: 'TWEET.V1',
  payload: {
    text: string
    images: string[]
  }
}
```

Where:

- `text` is the templated body of the tweet.
- `images` is an array of links.

The text should be templated in the following way:

- Replace any soft line wrap with real line wraps. (this is necessary to have text on a new line in the spreadsheet without using `\n`) https://github.com/learnk8s/lms/blob/master/packages/smartqueue-importer/index.ts#L88
- You should combine the tweet with this logic: https://github.com/learnk8s/lms/blob/master/packages/smartqueue-importer/index.ts#L95-L108 Note how there are 2 variations. 

## Notes

- You will probably need to build an interface to upload files or copy-paste the content for a CSV file. [Node-red has an example](https://flows.nodered.org/flow/c70d0b4c54b583cf30c7e989b74feb68) but using this is problematic since it's hard to report errors in the CSV.
- If you need to build a custom UI, you should consider having a look at these notes: https://github.com/uasabi/noderosso#usual-and-unusual-use-cases
